pydecred/account.py

```diff
@@ -360,12 +360,6 @@ def purchaseTickets(self, qty, price):
                 self.addMempoolTx(tx)
         # Store the txids.
         self.tickets.extend([tx.txid() for tx in txs[1]])
-        # Remove spent utxos from cache.
-        self.spendUTXOs(spentUTXOs)
-        # Add new UTXOs to set. These may be replaced with network-sourced
-        # UTXOs once the wallet receives an update from the BlockChain.
-        for utxo in newUTXOs:
-            self.addUTXO(utxo)
```

Comment:
Waiting for the updates from dcrdata here instead. This should help with some of the transient balance display issues. 

The balance display being wrong for a moment at startup is because the balance is being stored as part of the encrypted wallet object, so when an update to the balance occurs while the wallet is locked, the new value is not properly stored. I have a plan, but it should wait until the database changes proposed in [the budget proposal](https://proposals.decred.org/proposals/ad0f9688b3467734e2581604914b2cc32c6eb7991dff460eff41d21f66d88451) are complete.